lib/dlss/capistrano/tasks/setup/setup.rake

```diff
@@ -0,0 +1,20 @@
+# Setup controlmaster SSH session before generating one-time-key
+before 'otk:generate', 'controlmaster:setup'
+
+DOES_NOT_REQUIRE_OTK_LIST = [
+  'bundler:map_bins',
+  'controlmaster:setup',
+  'controlmaster:start',
+  'default',
+  'deploy:generate_otk',
+  'install',
+  'load:defaults',
+  'otk:generate',
+  *Rake.application.stages
+].freeze
+
+Rake.application.tasks.each do |task|
+  next if DOES_NOT_REQUIRE_OTK_LIST.include?(task.name)
+
+  before task, 'deploy:generate_otk'
+end
```

Comment:
Translation: make sure all tasks that require one-time keys have them, and make sure we have a controlmaster in place before we generate one-time keys. Note that the controlmaster tasks are still opt-in, so that behavior should be unaffected here.